queue_services/entity-emailer/src/entity_emailer/email_templates/RES-limitedRestorationExtension-PAID.jinja2

```diff
@@ -0,0 +1,8 @@
+{% extends 'RES-PAID.jinja2' %}
+
+{% block attachments %}
+    <ul>
+        <li>Limited Restoration Extension Application</li>
+        <li>Receipt</li>
+    </ul>
+{% endblock %}
```

Comment:
When the ticket requirements changed and the business requested specific email wording for limited restoration extensions that have been paid, the only change required was to create a new jinja2 file with the correct name -- no change was required to the existing templates.  I think this shows the benefit of this template architecture as it satisfies the "open closed principle" from the SOLID design principles -- objects or entities should be open for extension but closed for modification.